Kickstarter-iOS/Features/ProjectPage/Controller/ProjectPageViewControllerTests.swift

```diff
@@ -44,11 +44,15 @@ internal final class ProjectPageViewControllerTests: TestCase {
           sourceURLString: "https://source.com",
           thumbnailURLString: nil,
           seekPosition: .zero
-        ),
-        ExternalSourceViewElement(
-          embeddedURLString: "https://source.com",
-          embeddedURLContentHeight: 123
         )
+        /*
+          We need a way to mock these if we want to test them.
+          For now, disabling the external web view test.
+         ExternalSourceViewElement(
+           embeddedURLString: "https://source.com",
+           embeddedURLContentHeight: 123
+         )
+         */
```

Comment:
This element was only _rarely_ showing up in the snapshots, so we weren't testing it effectively anyways. Disabling it will get the tests back running consistently for now.

If we want to test this more specifically, we could load a local file URL and pass that into the web view. But that requires some actual changes to the `ExternalSource` cell, and I'd rather just fix the test for now.